Runtime/Scripts/RGServiceManager.cs

```diff
@@ -512,6 +512,37 @@ public async Task UploadValidations(long botInstanceId, string filePath, Action
             }
         }
 
+        /**
+         * Uploads all log messages for the bot instance to Regression Games in JSONL format
+         */
+        public async Task UploadLogs(long botInstanceId, string filePath, Action onSuccess, Action onFailure)
+        {
+            if (await EnsureAuthed())
+            {
+                await SendWebFileUploadRequest(
+                    uri: $"{GetRgServiceBaseUri()}/bot-instance-history/{botInstanceId}/logs",
+                    method: "POST",
+                    filePath: filePath,
+                    contentType: "application/jsonl",
+                    onSuccess: async (s) =>
+                    {
+                        RGDebug.LogDebug(
+                            $"RGService UploadLogs response received");
+                        onSuccess.Invoke();
+                    },
+                    onFailure: async (f) =>
+                    {
+                        RGDebug.LogWarning($"Failed to upload logs for bot instance {botInstanceId}: {f}");
+                        onFailure.Invoke();
+                    }
```

Comment:
Interestingly, I did notice this issue (it occurs with our existing validations upload). Fortunately, if the file is missing, it produces a warning rather than throwing :).